main.go

```diff
@@ -110,6 +116,33 @@ sfdeploy.serverurl = https://login.salesforce.com
 sfdeploy.maxPoll = 20`
 
 func main() {
+	var projectPath string
+	flag.Parse()
+
+	if *dFlag {
```

Comment:
I would switch the `-d` flag to parse a string.  Then, you could avoid the prompt for user input during execution.  `go-sf-ant -d /Users/me/Documents`.  Totes up to you.

Reply:
Oh I didn't know you can execute this way, I always though it was always `go run main.go`. Okay so, 101 time. When you `install` or `go get` it adds `go-sf-ant` as a command?

Reply:
Correct.  `go run ...` is just a quick way to run your program.  It compiles, runs it, then destroys the binary that it compiled.  `go build` compiles your program and produces the binary.  Think of `go install` as `go build` + `move to $GOPATH/bin`.